### ISSUE TITLE
Uncategorised method in StDebuggerCommand>>#transform:

### ISSUE THREAD

**Issue by MarcusDenker** [pharo-spec/NewTools#377]
StDebuggerCommand>>#transform: should be categorised

**Comment by MarcusDenker** [comment]
Do we need the method?

```
transform: aBlock

	^ self

```

There seems to be one sender in the debugger command that sets a block?


(this shows again that the idea that uncategorized methods can be fixed later is wrong: I have *NO IDEA* what a good categeory could be)

**Comment by MarcusDenker** [comment]
I added the class as an exception to testNoUncategorizedMethods for now, as soon as this Issue is fixed we can remove it.

**Comment by adri09070** [comment]
I remember this method being introduced by this PR #373, in which I contributed during the sprint 2 months ago with @StevenCostiou  and @maxwills.

From what I remember, `transform:` used to transform a command into a spec command. However, Spec changed its API, which broke the stack commands in the debugger. So I think #373 changed the context of the spec command to the debugger itself not to rely on Spec anymore.

So, at first, we tried to implement a `transform:` method but I think that it was useless and that it shouldn't have been committed. At least, I tried on an image to remove the `transform:` method and to delete the only line of code that calls it in 
```Smalltalk
StDebuggerStackCommandTreeBuilder>>#buildSpecCommand:forContext:
```
and the stack commands still work and all tests pass.

I'm not so sure though, but I know that @StevenCostiou knows.
